src/java/org/apache/cassandra/config/Config.java

```diff
@@ -392,19 +454,28 @@
     /**
      * Size of the CQL prepared statements cache in MB.
      * Defaults to 1/256th of the heap size or 10MB, whichever is greater.
+     *
      */
-    public Long prepared_statements_cache_size_mb = null;
+    @Replaces(oldName = "prepared_statements_cache_size_mb", converter = Converter.MegabytesDataStorageConverter.class, deprecated = true)
+    public DataStorage prepared_statements_cache_size = null;
+
+    @Replaces(oldName = "enable_user_defined_functions", deprecated = true)
+    public boolean user_defined_functions_enabled = false;
 
-    public boolean enable_user_defined_functions = false;
-    public boolean enable_scripted_user_defined_functions = false;
+    @Replaces(oldName = "enable_scripted_user_defined_functions", deprecated = true)
+    public boolean scripted_user_defined_functions_enabled = false;
 
-    public boolean enable_materialized_views = false;
+    @Replaces(oldName = "enable_materialized_views", deprecated = true)
+    public boolean materialized_views_enabled = false;
 
-    public boolean enable_transient_replication = false;
+    @Replaces(oldName = "enable_transient_replication", deprecated = true)
+    public boolean transient_replication_enabled = false;
 
-    public boolean enable_sasi_indexes = false;
+    @Replaces(oldName = "enable_sasi_indexes", deprecated = true)
+    public boolean sasi_indexes_enabled = false;
 
-    public volatile boolean enable_drop_compact_storage = false;
+    @Replaces(oldName = "enable_drop_compact_storage", deprecated = true)
```

Comment:
It looks like we should update in `doc/source/configuration/case_yaml_file.rst` all the cases where we've moved the `enabled` prefix to an `enabled` suffix. (`enable_materialized_views`, `enable_sasi_indexes`, etc.)

Reply:
With ascii doc